mypage/views.py

```diff
@@ -1,6 +1,75 @@
+import json
 from django.views import View
+from datetime import datetime
+from django.utils.dateformat import DateFormat
+from django.forms.models import model_to_dict
+from urllib import parse
 from django.http import JsonResponse,HttpResponse
-import json
+from django.db import models
+from django.utils import timezone
+
+from django.db.models import F, Sum, Count, Case, When
+
+from .models import WishesDetail, CartDetail
+from user.models import User, UserType
+from product.models import (
+    SocksCategory,
+    SocksType,
+    PatternType,
+    PatternDescription,
+    LogoType,
+    LogoDescription,
+    DesignDescription
+    )
 
 def mypage_test(request):
-    return HttpResponse("<h1>테스트</h1>")
\ No newline at end of file
+    return HttpResponse("<h1>테스트</h1>")
+
+class MyWishesView(View):
+
+    def post(self, request):
+
+        wish_list_req   = json.loads(request.body)
+        my_wish_list    = {
+            'user_pk'         : wish_list_req["user_pk"],
+            'my_wish_list'  :[{
+                'id'            :   wishes_detail["id"],
+                'my_wish_label' :   wishes_detail["label"],
+                'wished_date'   :   wishes_detail["created_at"],
+                'design'        :   model_to_dict(DesignDescription.objects.get(id = wishes_detail["design_id"]))
+                }for wishes_detail in WishesDetail.objects.filter(user_id = wish_list_req["user_pk"]).values()]
```

Comment:
`} for wishes_detail in WishesDetail.objects.filter(user_id = wish_list_req["user_pk"]).values()]`